src/Sulu/Bundle/WebsiteBundle/Routing/ContentRouteProvider.php

```diff
@@ -66,12 +66,13 @@ public function getRouteCollectionForRequest(Request $request)
 
         if ($this->requestAnalyzer->getMatchType() == RequestAnalyzerInterface::MATCH_TYPE_REDIRECT
             || $this->requestAnalyzer->getMatchType() == RequestAnalyzerInterface::MATCH_TYPE_PARTIAL
+            || preg_match('/\/$/', $resourceLocator)
```

Comment:
There is already a `checkResourceLocator` function, please check if it is still needed.